app/assets/javascripts/members/activities/activities.js

```diff
@@ -49,7 +49,7 @@ function rgbToHex(rgb) {
 }
 
 function confirm_enroll(activity) {
-  swal({
+  Swal.fire({
```

Comment:
inschrijven /uitschrijven /etc. werkt niet meer

Ik krijg de volgende warnings in de browser console:
`SweetAlert2: Unknown parameter "type" sweetalert2.min.source.js:1
SweetAlert2: Unknown parameter "closeOnConfirm"`